### PR TITLE
optimize README

### PR DESCRIPTION
**Pull request by fm-sys**
- Improve description
- Add installation guide
- referencing to Tampermonkey as preferred user script manager (as on the [send2cgeo website](https://www.cgeo.org/send2cgeo.html))
- Fix typo

### CODE REVIEW

**Comment by Lineflyer** [comment]
I put the README.MD onto the ignore list on Codacy.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,16 @@
 # send2cgeo
 
-Script for Mozilla Firefox and Google Chrome browser to [send geocaches directly from your browser to c:geo](http://www.cgeo.org/send2cgeo.html) on your phone.
+**send2cgeo** is a Tampermonkey/Greasemonkey Script for geocaching.com and opencaching.de, which allows you to send geocaches directly from your browser to [c:geo](https://www.cgeo.org/) on your phone.
```
Comment (bekuno) [inline README.md:3]: change scriot to lowercase: `is a Tampermonkey/Greasemonkey script`

File: README.md [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
+For more information, please visit the [send2cgeo website](https://www.cgeo.org/send2cgeo.html).
+## Installation
+- Install the [Tampermonkey Add-On](https://www.tampermonkey.net/) or a different user script manager in your browser
+- After that, click [here](https://github.com/cgeo/send2cgeo/raw/release/send2cgeo.user.js) to install send2cgeo inside your user script manager (e.g. Tampermonkey)
+- Finally, you have to [pair your browser with your phone](https://www.cgeo.org/send2cgeo.html#registering-browser-and-device) and you're ready! 
+- If you need any further help, it's the best to visit the [send2cgeo installation guide](https://www.cgeo.org/send2cgeo#how-to-install-send2cgeo) at our website
```
Comment (bekuno) [inline README.md:9]: change to `on our website`

File: README.md [hunk 3]
```diff
@@ -1,9 +1,16 @@
...
 
-The script download link on the [c:geo homepage](http://cgeo.org) as well as the update mechanism for the installed script directly targets the raw script on the `release` branch, which means after merging the changes to `release`, together with the required version number increase, users will see the update on the website and script hosts (e.g. Greasemonkey) will detect and notify users to update.
+The script download link on the [c:geo homepage](http://cgeo.org) as well as the update mechanism for the installed script directly targets the raw script on the `release` branch, which means after merging the changes to `release`, together with the required version number increase, users will see the update on the website and script hosts (e.g. Tampermonkey) will detect and notify users to update.
```
Comment (bekuno) [inline README.md:16]: The officially homepage adress is `https://www.cgeo.org`.

**Comment by fm-sys** [comment]
I have adopted the changes suggested by @bekuno. Let's merge, further changes can be made afterwards if necessary...

**Pull request merged**